### DISCUSSION TITLE
Website sidebar menu issue

### DISCUSSION THREAD

**Discussion by rodgerwerner** [elrumo/macOS_Big_Sur_icons_replacements#1210] [Help with the website] [answered]
In Safari the side menu bar has this nasty gradient issue at the bottom:
<img width="258" alt="Screen Shot 2021-06-17 at 9 59 52 AM" src="https://user-images.githubusercontent.com/23433246/122411732-fafbb580-cf52-11eb-8454-92a823dd6686.png">

Is this being addressed @elrumo ?

**Comment by elrumo** [comment, marked as answer]
The latest commit fixes the issue, thanks!
